API.md

```diff
@@ -106,6 +118,8 @@ Returns a single election with some more data than the list:
 
 ## Voters API
 
+### `POST /api/mailelection/:electionId/mailinglist`
```

Comment:
Request / response for this?

Reply:
I was just thinking default promise resolved. What do you think?

Reply:
What is the endpoint for uploading a csv voter mailinglist file?